PRESUBMIT.py

```diff
@@ -286,7 +286,7 @@ def CheckLicense(input_api, output_api):
 
 _BANNED_CPP_FUNCTIONS += (
     BanRule(
-        r'/\bStringPiece',
+        r'/\b(Basic|W)?StringPiece(16)?\b',
```

Comment:
is this relevant to this reclient change?

Reply:
Yes, it's relevant to `redirect_cc` change in this PR. Without this presubmit complains on `base::FilePath::StringPieceType` usages.

Reply:
Yes, but I'm saying that might not be a desirable failure mode because even testing this will require people to delete/modify npm config to switch between them